snuba/snuba_migrations/spans/0007_spans_add_metrics_summary.py

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.SPANS
+local_table_name = "spans_local"
+dist_table_name = "spans_dist"
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Add metrics_summary column.
+    """
+
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=storage_set_name,
+                table_name=local_table_name,
+                column=Column(
+                    "metrics_summary",
```

Comment:
Ideally, since you don't want to search on these in the dataset, but just visualize them in a trace view, this column is good candidates for putting into a place like nodestore (similar to contexts). But since that is not available now, this should be ok.